saber.html

```diff
@@ -3,10 +3,48 @@
 
 <head>
     <title>Document</title>
+    <script>
+        function getData() {
+            const name = document.getElementById('my-name').value;
+
+            if (name == '') {
```

Comment:
You don't need this condition, because there's already "required" included in your form.